src/sentry/web/frontend/idp_email_verification.py

```diff
@@ -0,0 +1,20 @@
+from django.http.response import HttpResponseNotFound, HttpResponseRedirect
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.auth.idpmigration import get_redis_key, verify_account
+from sentry.utils.auth import get_login_url
+from sentry.web.frontend.base import BaseView
+
+
+class IDPEmailVerificationView(BaseView):
```

Comment:
since this a simple redirect i don't think you need to wrap the method in a view.